orderer/common/server/etcdraft_test.go

```diff
@@ -249,3 +265,35 @@ func launchOrderer(gt *GomegaWithT, orderer, tempDir, genesisBlockPath, fabricRo
 	gt.Expect(err).NotTo(HaveOccurred())
 	return sess
 }
+
+func restartOrderer(gt *GomegaWithT, orderer, tempDir, genesisBlockPath, cryptoPath string) *gexec.Session {
```

Comment:
there is a "launch orderer" method above this one. Can't we use it somehow? do we have to copy all the env vars?

Reply:
done